core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/hdr/InternalHDRPercentilesRanksTests.java

```diff
@@ -23,19 +23,21 @@
 import org.elasticsearch.common.io.stream.Writeable.Reader;
 import org.elasticsearch.search.DocValueFormat;
 import org.elasticsearch.search.aggregations.metrics.percentiles.InternalPercentilesRanksTestCase;
-import org.elasticsearch.search.aggregations.metrics.percentiles.ParsedPercentileRanks;
+import org.elasticsearch.search.aggregations.metrics.percentiles.ParsedPercentiles;
 import org.elasticsearch.search.aggregations.pipeline.PipelineAggregator;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 
 public class InternalHDRPercentilesRanksTests extends InternalPercentilesRanksTestCase<InternalHDRPercentileRanks> {
 
     @Override
     protected InternalHDRPercentileRanks createTestInstance(String name, List<PipelineAggregator> aggregators, Map<String, Object> metadata,
-                                                            double[] cdfValues, boolean keyed, DocValueFormat format) {
-        DoubleHistogram state = new DoubleHistogram(3);
-        return new InternalHDRPercentileRanks(name, cdfValues, state, keyed, format, aggregators, metadata);
+                                                            boolean keyed, DocValueFormat format, double[] percents, double[] values) {
+        final DoubleHistogram state = new DoubleHistogram(3);
+        Arrays.stream(values).forEach(state::recordValue);
```

Comment:
I was trying to understand why this step got necessary, can you briefly explain? I'm just curious what happens here since it didn't happen in the test before.